lib/pascals_triangle.rb

```diff
@@ -0,0 +1,30 @@
+class PascalsTriangle
+
+  def initialize
+    @triangle = [[1]]
+  end
+
+  def one_row(row_num)
+    if @triangle[row_num]
+      return @triangle[row_num]
+    end
+    previous_row = one_row(row_num - 1)
+    @triangle << calculate(previous_row)
+
+    @triangle[row_num]
+  end
+
+  def calculate(previous_row)
+    middle = []
+    (previous_row.length - 1).times do |sum|
```

Comment:
@StefanGajic `sum` is misleading here. This is an index in an array, it's not the sum of anything.